nima_core/precognition/retrospective_scanner.py

```
@@ -0,0 +1,218 @@
+    "from the provided memory excerpts. "
+    "Return ONLY a JSON array — no prose, no markdown fences."
+)
+
+_USER_TEMPLATE = """\
+Review the following memory excerpts and identify items that require follow-up.
+For each item return a JSON object with keys:
+  "type"         : "task" | "project" | "person" | "issue" | "decision"
+  "subject"      : concise label (max 80 chars)
+  "summary"      : one-sentence explanation of what is unresolved
+  "last_seen_days": approximate days since last mentioned (integer)
+  "urgency"      : float 0.0–1.0 (higher = more urgent)
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)